tests/specs/session-zip.e2e.ts

```diff
@@ -0,0 +1,68 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import JSZip from 'jszip';
+import { volViewPage } from '../pageobjects/volview.page';
+import { TEMP_DIR } from '../../wdio.shared.conf';
+
+// from https://stackoverflow.com/a/47764403
+function waitForFileExists(filePath: string, timeout: number) {
+  return new Promise<void>((resolve, reject) => {
+    const dir = path.dirname(filePath);
+    const basename = path.basename(filePath);
+    let timerId = undefined as NodeJS.Timeout | undefined;
+    const watcher = fs.watch(dir, (eventType: any, filename: string | null) => {
+      if (eventType === 'rename' && filename === basename) {
+        clearTimeout(timerId);
+        watcher.close();
+        resolve();
+      }
+    });
+
+    const onTimeout = () => {
+      watcher.close();
+      reject(
+        new Error('File did not exists and was not created during the timeout.')
+      );
+    };
+
+    timerId = setTimeout(onTimeout, timeout);
+
+    fs.access(filePath, fs.constants.R_OK, (err) => {
+      if (!err) {
+        clearTimeout(timerId);
+        watcher.close();
+        resolve();
+      }
+    });
+  });
+}
+
+describe('VolView', () => {
+  beforeEach(async () => {
+    // cleanup temp directory
```

Comment:
Is this a todo?

Reply:
Ooh, zombie code.  Went with the `cleanuptotal` service instead.